go.mod

```diff
@@ -23,6 +23,8 @@ require (
 	github.com/prometheus/client_golang v1.0.0
 	github.com/smartystreets/assertions v1.0.1 // indirect
 	github.com/smartystreets/goconvey v0.0.0-20190731233626-505e41936337
+	github.com/tidwall/gjson v1.4.0 // indirect
+	github.com/tidwall/sjson v1.0.4
```

Comment:
@primalmotion @t00f ℹ️This package allows us to CRUD raw JSON efficiently without having to incur the penalty of (un)marshaling again.

This is useful for injecting the push configuration ID after having encoding the event once for all clients.